### PR TITLE
Rc/3.39.1

### PR DESCRIPTION
**Pull request by daniel-cy-lu**
**Description of changes**

Rc/3.39.1

**Type of Change**

- [ ] QA release

#508 Extend API to include some program metadata
PR 658

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
 		"eslint-plugin-import": "^2.27.5",
 		"eslint-plugin-prettier": "^4.2.1",
 		"jest": "^25.5.4",
-		"nodemon": "^1.19.4",
+		"nodemon": "^2.0.22",
```
Comment (Buwujiu) [inline package.json:92]: why is this changed?

Comment (daniel-cy-lu) [reply package.json:92]: Screenshot 2023-05-23 at 4 04 00 PM

It's from Dan's ticket when he ran a npm audit fix two weeks ago.

Comment (daniel-cy-lu) [reply package.json:92]: @demariadaniel Dan, can you take a look at the change to nodemon version update, before we merge this to main? Thank you

Comment (demariadaniel) [reply package.json:92]: Yes this is patching a security vulnerability

**Review by demariadaniel — APPROVED** [review]

**Comment by daniel-cy-lu** [comment]
@demariadaniel Thanks Dan!
